recipes/prismatic/meta.yaml

```
@@ -0,0 +1,164 @@
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - zlib
+      run:
+        - zlib
+        - hdf5
+        - fftw
+
+
+    test:
+      source_files:
+        - SI100.XYZ
+      commands:
+        - prismatic -i SI100.XYZ
+
+
+  - name: prismatic_gui
+    build:
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh gui # [not win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - qt 5.9.*
+        - h5py
+        - zlib
+      run:
+        - zlib
+      
+    test:
+      commands:
+        - command -v prismatic-gui  # [linux]
+        - test -f $PREFIX/Applications/prismatic-gui.app/Contents/MacOS/prismatic-gui  # [osx]
+
+  - name: pyprismatic
+    build:
+      skip: True  # [win or py2k or osx]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh py # [not win]
+
+    requirements:
+      build:
+        - python
+        - pip
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - python
+        - boost
+        - hdf5
+        - fftw
+        - qt 5.9.*
+        - h5py
+        - zlib
+      run:
+        - python
+        - numpy
+        - h5py
+        - zlib
+
+    test:
+      commands:
+        - python -c "import pyprismatic; pyprismatic.demo()"
+
+about:
+  home: http://prism-em.com/
+  license: GNU
```

Which licence is it? Use the SPDX identifier https://spdx.org/licenses/
I think it should be
```suggestion
  license:  GPL-3.0-only
```

---

It is GPL 3 - I'll apply this change.